pyartcd/pyartcd/pipelines/operator_sdk_sync.py

```diff
@@ -0,0 +1,133 @@
+import os
+import re
+import subprocess
+
+import click
+import koji
+import yaml
+from errata_tool import Erratum
+from pyartcd import constants
+from pyartcd.cli import cli, click_coroutine, pass_runtime
+from pyartcd.runtime import Runtime
+
+
+class OperatorSDKPipeline:
+    def __init__(self, runtime: Runtime, group: str, assembly: str, updatelatest: bool) -> None:
+        self.runtime = runtime
+        self._logger = runtime.logger
+        self.version = group.split("-")[1]
+        self.assembly = assembly
+        self.updatelatest = updatelatest
+        self.sdk = "operator-sdk"
+
+        self._jira_client = runtime.new_jira_client()
+        ocp_build_data_repo = runtime.new_github_client().get_repo("openshift/ocp-build-data")
+        release_file = ocp_build_data_repo.get_contents("releases.yml", ref=group)
+        release_yaml = yaml.load(release_file.decoded_content, Loader=yaml.FullLoader)
+        self.extra_ad_id, self.parent_jira_key = self.get_ad_jira_key(assembly, release_yaml)
+
+    def run(self):
+        advisory = Erratum(errata_id=self.extra_ad_id)
+        self._logger.info("Check advisory status ...")
+        if advisory.errata_state in ["QE", "NEW_FILES"]:
+            self._logger.info("Advisory status not in REL_PREP yet ...")
+            return
+        if advisory.errata_state == "SHIPPED_LIVE":
+            self._logger.info("Advisory status already in SHIPPED_LIVE, update subtask 9 ...")
+            self._update_jira(self.parent_jira_key, 8, "Advisory status already in SHIPPED_LIVE")
+        self._logger.info("Advisory status already in post REL_PREP, update subtask 7 ...")
+        self._update_jira(self.parent_jira_key, 6, "Advisory status already in REL_PREP")
+
+        et_builds = advisory.errata_builds
+        sdk_build = [i for i in et_builds[f'OSE-{self.version}-RHEL-8']
+                     if re.search("openshift-enterprise-operator-sdk-container*", i)]
+        if not sdk_build:
+            self._logger.info("No SDK build to ship, update subtask 8 then close ...")
+            self._update_jira(self.parent_jira_key, 7,
+                              f"No SDK build to ship, operator_sdk_sync job: {os.environ.get('BUILD_URL')}")
+            return
+
+        build = koji.ClientSession(constants.BREW_SERVER).getBuild(sdk_build[0])
+        archlist = ["amd64", "arm64", "ppc64le", "s390x"]
```

Comment:
`archlist` can be get from ImageMetadata, or we can just query it through Brew buildInfo API.

Reply:
this job is for operator-sdk usage, sdk should support full arch

Reply:
I wouldn't assume that. In case we only built a few arches, or the arch list changes in the future, we may forget to update here.
If you want to keep this simple, I would recommend to keep this a build parameter with default value of the current archlist.